CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## TBD
+
+* Delete tombstoned files left in cache directory
```

Comment:
This entry should explain what this change means for crash reporting and why it might happen/be necessary to do this.